### DISCUSSION TITLE
Help Wanted

### DISCUSSION THREAD

**Discussion by xfqwdsj** [xfqwdsj/IAmNotADeveloper#30] [Announcements]
It has now been discovered that adding "System Framework" to the LSPosed scope will not prevent the system from booting. The advantage of this is that the application will not detect that it has been tampered with by Xposed. I need you to help me test the module (using version 1.1.0: <https://github.com/Xposed-Modules-Repo/xyz.xfqlittlefan.notdeveloper/releases/tag/4-1.1.0>) while keeping your device safe (you can recover your device yourself) and provide me with the following feedback:

- Device Android version (if you are using a custom system, please indicate the custom system and its version)
- LSPosed version
- How the module works (e.g. if the module is enabled only for the "System Framework" or if you need to check the corresponding apps, or if it doesn't work at all)

If you have any other information to add, please feel free to do so. Thank you for your participation!

**Comment by rushiranpise** [comment]
Device Samsung A50 Oneui 4
Latest Lsposed CI build, kernelsu
Added only system framework to scope & reboot
No bootloop
I have turned on developer mode, adb both wifi and usb

The app shows it's turned off.

Yet ruru can detect it , unless added to scope (still detect adb though)
Haven't tested it with any other app , since I don't know which app detect adb specifically, never encountered such app

![Screenshot_20231016-111830_Shell](https://github.com/xfqwdsj/IAmNotADeveloper/assets/37355997/c4d550ef-beb0-4159-8fc6-88fe4063a356)
![Screenshot_20231016-111856_Ruru](https://github.com/xfqwdsj/IAmNotADeveloper/assets/37355997/6aebaf8c-f9a9-4f92-aa19-8438bee3f7bd)
![Screenshot_20231016-111937_IAmNotADeveloper](https://github.com/xfqwdsj/IAmNotADeveloper/assets/37355997/1ed55c88-f3c9-4ca9-99dc-5dba8ed91fd0)

**Reply by xfqwdsj** [reply]
Thank you!

**Comment by rushiranpise** [comment]
After adding ruru to scope, same ruru with/without system framework in scope
![Screenshot_20231016-112835_Ruru](https://github.com/xfqwdsj/IAmNotADeveloper/assets/37355997/379333ec-5d64-4369-a1c3-d3340d7d9423)

**Reply by xfqwdsj** [reply]
A version with (incomplete) props hidding has not yet been released and it's another question. Please review https://github.com/xfqwdsj/IAmNotADeveloper/commit/b3c4786e164b36ffc2b7ad5287a3b1c98f3abb29 and check this version packaged from ci.

**Reply by rushiranpise** [reply]
yes using same

**Comment by rushiranpise** [comment]
xposed log dont show the ruru app prop being changed,maybe the hook is not working

**Reply by xfqwdsj** [reply]
Ruru uses process to get props, but we doesn't handle it.

**Reply by rushiranpise** [reply]
I think i have found a solution

**Reply by rushiranpise** [reply]
Flash this module and reboot,with ruru in module scope
<img src="https://github.com/xfqwdsj/IAmNotADeveloper/assets/37355997/07d5f43e-2e92-49b1-9c1e-0541bd99bb5f" width="200" height="400">

[Hide Debugging.zip](https://github.com/xfqwdsj/IAmNotADeveloper/files/13110843/Hide.Debugging.zip)

**Reply by rushiranpise** [reply]
Download from here https://github.com/rushiranpise/Hide-Debugging/releases/download/1.0/Hide.Debugging.1.0.zip

**Reply by privacyguy123** [reply]
Momo still detecting Debugging which means other banking apps that use this method probably are too - works in Ruru though even though my app looks different than yours. How do you hide HMA from Ruru? 

![image](https://github.com/xfqwdsj/IAmNotADeveloper/assets/50337995/3eee78cf-6056-4bb2-9d7c-4d848d9fefa7)

Seems to add small lag to all settings window opening too

**Reply by rushiranpise** [reply]
momo is on another level, cant hide it from momo

**Reply by privacyguy123** [reply]
Some are able to hide zygisk from Momo, I have not figured it out yet - USB debugging should be simple compared to that 👀
